CHANGES.md

```diff
@@ -1,3 +1,9 @@
+next
+----
+
+- Fix an `exception End_of_file` crash that could happen on Windows
```

Comment:
Note that without the corresponding fixes to cram_exec, this PR doesn't actually change anything.

Reply:
Ok, I removed the changelog entry. I expected it would be possible to trigger, but if nobody has seen it then perhaps not.